images/sk-opts.proto

```diff
@@ -26,13 +26,10 @@ message sk_opts_entry {
 	optional bool		so_reuseport	= 17;
 	optional bool		so_broadcast	= 18;
 	optional bool		so_keepalive	= 19;
-	optional uint32		tcp_keepcnt	= 20;
-	optional uint32		tcp_keepidle	= 21;
-	optional uint32		tcp_keepintvl	= 22;
-	optional uint32		so_oobinline	= 23;
-	optional uint32		so_linger	= 24;
+	optional uint32		so_oobinline	= 20;
+	optional uint32		so_linger	= 21;
 
-	optional uint32		so_buf_lock	= 25;
+	optional uint32		so_buf_lock	= 22;
```

Comment:
@juntongdeng These changes would break backward compatibility.
https://criu.org/Postulates#Protobuf_images_compatibility

Reply:
Sorry, I will fix it

Reply:
fixed